app.js

```diff
@@ -43,7 +46,28 @@ app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(cookieParser());
 app.use(compression());
-app.use(express.static(path.join(__dirname, 'public')));
+app.use(express.static(publicDirectory));
+
+if (app.get('env') !== 'development') {
+  var livereload = require('livereload');
```

Comment:
For these new `require` statements, `const` would be better than `var`. The old code was written before JavaScript had `const`, but you can use `const` for this new code.

Reply:
Good to know.  I was hesitating on using `const`  or keeping consistency across the file. But using `const` is a much better approach.